src/org/opensolaris/opengrok/authorization/AuthorizationFramework.java

```diff
@@ -141,6 +151,38 @@ public void setPluginDirectory(String directory) {
         setPluginDirectory(directory != null ? new File(directory) : null);
     }
 
+    /**
+     * Make {@code reload()} search for plugins in class files.
+     * @param flag true or false
+     */
+    public void setLoadClasses(boolean flag) {
+        loadClasses = flag;
+    }
+    
+    /**
+     * Whether to search for plugins in class files.
+     * @return true if enabled, false otherwise
+     */
+    public boolean getLoadClasses() {
```

Comment:
better naming for boolean property getter?

Reply:
fixed